plugins/invisible-typing/src/index.ts

```diff
@@ -12,7 +12,7 @@ export interface SettingsType {
   button?: boolean;
   invisible?: boolean;
   channelWise?: boolean;
-  channels?: Record<string, boolean>;
+  channels?: Record<string, boolean | undefined>;
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Allowing 'undefined' in channel settings type may introduce ambiguity.

Consider if a more explicit approach to handle non-existent channels could be beneficial for clarity and maintainability.

```suggestion
  channels?: Record<string, boolean | null>;
```